components/admin_console/billing/billing_summary/billing_summary.tsx

```diff
@@ -172,7 +209,7 @@ export const lastInvoiceInfo = (invoice: any, product: any, fullCharges: any, pa
                     className='BillingSummary__lastInvoice-charge'
                 >
                     <div className='BillingSummary__lastInvoice-chargeDescription'>
-                        {product.billing_schema === BillingSchemes.FLAT_FEE ?
+                        {product?.billing_scheme === BillingSchemes.FLAT_FEE ?
```

Comment:
Oh interesting. I wonder why TypeScript didn't catch this before.

Reply:
The product was a function arg with `any` type. I changed it to a component and gave it a proper type